packages/client/test/integration/mocks/mockchain.ts

```diff
@@ -29,7 +29,6 @@ export default class MockChain extends Chain {
         {
           header: {
             number: number + 1,
-            difficulty: 1,
```

Comment:
is this only being used for PoS blocks?

Reply:
No, we have other integration tests that use the `mockChain`.  We don't have any reason to hard code this value though since the Block constructor will check the hardfork and give us a reasonable default (i.e. 0 if the consensus in `common` is PoS